docs/operation_semantics.md

```diff
@@ -1212,9 +1212,12 @@ A set of element-wise binary arithmetic operations is supported.
 
 <b> `Op(lhs, rhs)` </b>
 
-Where `Op` is one of `Add` (addition), `Sub` (subtraction), `Mul`
-(multiplication), `Div` (division), `Rem` (remainder), `Max` (maximum), `Min`
-(minimum), `LogicalAnd` (logical AND), or `LogicalOr` (logical OR).
+Where `Op` is one of `Add` (addition), `Atan2` (arc tangent), `Sub`
+(subtraction), `Mul` (multiplication), `Div` (division), `Pow` (power), `Rem`
+(remainder), `Max` (maximum), `Min` (minimum), `LogicalAnd` (logical AND),
+`LogicalOr` (logical OR), `LogicalXor` (logical XOR), `Complex` (complex),
```

Comment:
"complex" is a bit vague. I would change the description in parenthesis to: "Create complex number from two reals" or similar.

Also probably move Complex to end, keep all the bitwise ops (LogicalAnd through ShiftRightLogical) together.

Reply:
Moved Complex (combines real and imaginary parts into a complex number) to the end